velox/common/memory/Memory.h

```diff
@@ -157,6 +157,11 @@ class MemoryManager {
   /// 'incrementBytes'. The function returns true on success, otherwise false.
   bool growPool(MemoryPool* pool, uint64_t incrementBytes);
 
+  /// Invoked to shrink currently alive pools by 'targetBytes', by reducing this
```

Comment:
Remove "reducing this"? we can trigger memory reclaiming?

Consider
```
/// Invoked to shrink alive pools to free at least 'targetBytes' capacity. The function returns the actual freed memory capacity in bytes.
```

Reply:
I removed 'at least' because there might not be enough available memory to free to 'targetBytes' so the actual bytes freed might be smaller than 'targetBytes'. Also I think this is arbitrator implementation dependent so leaving more flexibility in words.